src/routes/add-place.ts

```diff
@@ -74,10 +74,8 @@ export default async function addPlace(fastify: FastifyInstance) {
           },
         });
       }
-
-      // back to places list
-      resp.header('HX-Redirect', `/`);
-      return;
+      resp.header('HX-Trigger-After-Swap', `GotoList`);
+      return `<div class="control"><progress class="progress is-small is-primary" max="100"></progress></div><span><i>processing</i></span>`;
```

Comment:
i don't think we should do this kinda thing. it is too easy to make a change in the progress-bar view and not know you need to update this. 

can we find another way? maybe use something like a view with `hx-swap-oob` instead?